migrations/versions/rev-2021-01-09-18:31:24-cd8488c19b16_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
+
+Revision ID: cd8488c19b16
+Revises: fb8dd6aed9a9
+Create Date: 2021-01-09 18:31:24.557692
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'cd8488c19b16'
+down_revision = 'fb8dd6aed9a9'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events', sa.Column('is_demoted', sa.Boolean(), nullable=False))
+    op.add_column('events_version', sa.Column('is_demoted', sa.Boolean(), autoincrement=False, nullable=True))
+    op.execute("UPDATE events SET is_demoted=False")
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('events_version', 'is_demoted')
+    op.drop_column('events', 'is_demoted')
+    op.execute("UPDATE events SET is_demoted=True")
```

Comment:
If you drop the column is_demoted, how can you set it to True? And even if you could, would you make all events demoted? Now, no event is being shown on frontend, WOW

Please be careful and think about these things :/

Reply:
I thought about it. But you have said before once that downgrade should be opposite of upgrade 😕